pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -129,11 +129,60 @@ func TestDeleteObservationLog(t *testing.T) {
 }
 
 func TestGetDbName(t *testing.T) {
-	// dbName := "root:@tcp(katib-mysql:3306)/katib?timeout=5s"
-	dbName := "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=disable"
-
-	if getDbName() != dbName {
-		t.Errorf("getDbName returns wrong value %v", getDbName())
+	cases := map[string]struct {
+		updateEnvs map[string]string
+		wantName   string
+	}{
+		"All parameters are default": {
+			updateEnvs: map[string]string{},
+			wantName:   "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=disable",
+		},
+		"Set DB_USER": {
+			updateEnvs: map[string]string{
+				common.DBUserEnvName: "testUser",
+			},
+			wantName: "host=katib-postgres port=5432 user=testUser password= dbname=katib sslmode=disable",
+		},
+		"Set KATIB_POSTGRESQL_DB_HOST": {
+			updateEnvs: map[string]string{
+				common.PostgreSQLDBHostEnvName: "testHost",
+			},
+			wantName: "host=testHost port=5432 user=katib password= dbname=katib sslmode=disable",
+		},
+		"Set KATIB_POSTGRESQL_DB_PORT": {
+			updateEnvs: map[string]string{
+				common.PostgreSQLDBPortEnvName: "1234",
+			},
+			wantName: "host=katib-postgres port=1234 user=katib password= dbname=katib sslmode=disable",
+		},
+		"Set KATIB_POSTGRESQL_DB_DATABASE": {
+			updateEnvs: map[string]string{
+				common.PostgreSQLDatabase: "testDB",
+			},
+			wantName: "host=katib-postgres port=5432 user=katib password= dbname=testDB sslmode=disable",
+		},
+		"Set DB_PASSWORD": {
+			updateEnvs: map[string]string{
+				common.DBPasswordEnvName: "testPassword",
+			},
+			wantName: "host=katib-postgres port=5432 user=katib password=testPassword dbname=katib sslmode=disable",
+		},
+		"Set KATIB_POSTGRESQL_SSL_MODE": {
+			updateEnvs: map[string]string{
+				common.PostgreSSLMode: "require",
+			},
+			wantName: "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=require",
+		},
 	}
 
+	for name, tc := range cases {
+		t.Run(name, func(t *testing.T) {
+			for k, v := range tc.updateEnvs {
+				t.Setenv(k, v)
+			}
+			if got := getDbName(); got != tc.wantName {
+				t.Errorf("getDbName() = %v, want %v", got, tc.wantName)
+			}
```

Comment:
```suggestion
			gotName := getDbName()
			if diff := cmp.Diff(tc.wantName, gotName); len(diff) != 0 {
				t.Errorf("Unexpected DBName (-want,+got):\n%s", diff)
			}
		
```

To improve visibility, we should use the `cmp.Diff`.

Reply:
Sure, update to `cmp.Diff`.